src/Microsoft.TestPlatform.AdapterUtilities/Helpers/ReflectionHelpers.cs

```diff
@@ -77,7 +77,7 @@ private static int ParseEscapedStringSegment(string escapedStringSegment, int po
                         else
                         {
                             stringBuilder.Append(escapedStringSegment[i + 1]);
```

Comment:
```suggestion
                            stringBuilder.Append(escapedStringSegment[++i]);
```

Reply:
This changes the behavior. The first one is side-effect free, while the other one increments the `i` on every call.

Reply:
I take it back. You can do multiline suggestions btw :)

Reply:
changed to suggestion